mono/metadata/threadpool-worker-default.c

```diff
@@ -571,6 +571,10 @@ worker_try_create (void)
 		return FALSE;
 	}
 
+#ifndef DISABLE_PERFCOUNTERS
```

Comment:
Much nicer and more efficient on native side.

Reply:
CAS on managed is equally efficient.